# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/gc/bug-1792338.js
# Commit: 453aa2faad97
# Full Hash: 453aa2faad97ce68b33bef75cd232e37d759556e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-29 09:50:10
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r d33708e4f8ec -r 453aa2faad97 js/src/jit-test/tests/gc/bug-1792338.js
--- a/js/src/jit-test/tests/gc/bug-1792338.js	Fri Oct 28 15:17:43 2022 +0000
+++ b/js/src/jit-test/tests/gc/bug-1792338.js	Fri Oct 28 15:17:43 2022 +0000
@@ -9,6 +9,6 @@
 enqueueMark(newGlobal({newCompartment: true}));
 enqueueMark('set-color-black');
 enqueueMark({});
-gcparam("markStackLimit", 1);
+setMarkStackLimit(1);
 gc();
 gc();